src/main/java/uk/gov/hmcts/reform/ccd/service/remote/DisposeCaseRemoteOperation.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.hmcts.reform.ccd.service.remote;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.data.em.CaseDocumentsDeletionResults;
+import uk.gov.hmcts.reform.ccd.data.em.DocumentsDeletePostRequest;
+import uk.gov.hmcts.reform.ccd.exception.DocumentDeletionException;
+import uk.gov.hmcts.reform.ccd.parameter.ParameterResolver;
+import uk.gov.hmcts.reform.ccd.util.SecurityUtil;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+@Service
+@Slf4j
+@Qualifier("DisposeCaseRemoteOperation")
+public class DisposeCaseRemoteOperation implements DisposeRemoteOperation {
```

Comment:
I'm not sure what's the purpose of DisposeRemoteOperation interface given that it's used in a single place